l10n_ch_pain_direct_debit/demo/dd_demo.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file should not be included in this PR, it was already done in branch https://github.com/CompassionCH/l10n-switzerland/tree/12.0-pain-direct-debit